examples/experimental/dagster-airlift/dagster_airlift/in_airflow/task_proxy_operator.py

```diff
@@ -63,7 +63,14 @@ def instantiate_dagster_operator(
     base_operator_args, base_operator_args_with_defaults = get_params(BaseOperator.__init__)
     init_kwargs = {}
 
-    ignore_args = ["kwargs", "args", "dag"]
+    ignore_args = [
+        # These don't make sense in context to copy
+        "kwargs",
+        "args",
+        "dag",
+        # The weight rule stored on the base operator is a different object than the constructor expects, and cannot be copied back to the constructor.
```

Comment:
Can you provide more context here? What are the types involved? Why can't it be copied etc?

Reply:
Added more detail and a permalink.